apps/explorer/src/utils/getAmount.ts

```diff
@@ -72,27 +74,26 @@ export function getTransfersAmount(
             : null;
     }
 
-    const paySuiData =
-        getPaySuiTransaction(txnData) ?? getPayTransaction(txnData);
+    const payData = getPaySuiTransaction(txnData) ?? getPayTransaction(txnData);
 
-    const amountByRecipient = paySuiData?.recipients.reduce(
+    const amountByRecipient = payData?.recipients.reduce(
         (acc, value, index) => {
```

Comment:
value is the address of the recipient right? can we rename it to something like `recipientAddress`?